charts/akeyless-api-gateway/templates/tls.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if ( and .Values.cache .Values.cache.tls .Values.cache.tls.autoGenerated) }}
+{{- $fullname := .Release.Name | trunc 63 | trimSuffix "-" -}}
+{{- $ca := genCA "redis-ca" 365 }}
+{{- $altNames := list $fullname "127.0.0.1" "localhost" (include "cache_address" .) (printf "*.%s" (include "cache_address" . ))}}
```

Comment:
remove localhost... we dont support external redis only the one we create here

Reply:
Localhost is not relevant to external redis. This certificate is only relevant for use in this cluster only

Reply:
we don't allow external redis! please remove

Reply:
Again, this has no connection to external redis. This certificate is only relevant for use in this cluster only

Reply:
Removed "127.0.0.1" "localhost" 
The customer can still access the cache if they exec into the redis pod even though "127.0.0.1" "localhost" were removed as per my tests